app/views/participants/_summary.html.haml

```diff
@@ -0,0 +1,11 @@
+%li.participant-summary.participant-summary--with-thumbnail
+  = link_to participant_url(participant), class: 'participant-summary__url' do
+    - if participant.photo.present?
```

Comment:
I tried to implement it but cannot do that because not familiar with CSS (or daimon_news-layout) . Can you help me with modifying `_summary.scss`?